runatlantis.io/docs/using-atlantis.md

```diff
@@ -124,6 +124,36 @@ Because Atlantis under the hood is running `terraform apply plan.tfplan`, any Te
 They're ignored because they can't be specified for an already generated planfile.
 If you would like to specify these flags, do it while running `atlantis plan`.
 
+---
+## atlantis import
```

Comment:
Could you also update the tfvars section on the custom workflows page to show how to set it up for the import step? I've run into issues before where i need to pass in a custom tfvars file in order to get the import command to run correctly

https://www.runatlantis.io/docs/custom-workflows.html#tfvars-files

Reply:
A test with tfvars may be useful too. Ignore me if that is overkill.

Reply:
same with https://github.com/runatlantis/atlantis/pull/2783#discussion_r1054818623